stories/jumbotron.stories.js

```diff
@@ -26,4 +26,31 @@ function JumbotronExample1() {
   );
 }
 
-export default JumbotronExample1;
+function JumbotronFluid() {
+  return (
+    <Jumbotron fluid>
+      <h1 className="display-3">
+        Hello, world!
+      </h1>
+      <p className="lead">
```

Comment:
You have to use the variable from emotion-bootstrap, classnames are changed when you compile bootstrap to js. So lead won't do anything. It might working if Storybooks is include bootstrap automatically, but its not the correct way to add the className.